vocab-builder-core/src/main/java/com/github/tkurz/sesame/vocab/VocabBuilder.java

```diff
@@ -296,6 +380,10 @@ public void generateResourceBundle(String baseName, Path bundleDir) throws Gener
     }
 
     public HashMap<String, Properties> generateResourceBundle(String baseName) throws GenerationException {
+    	// be sure to have at least the default URI constant settings
+    	if ( uriGeneration == null ) {
```

Comment:
The code style we have used here so far is not to have spaces inside of the if statement brackets. Ie, change to:

    if (uriGeneration == null) {